doc/guide/sso.xml

```diff
@@ -28,8 +28,11 @@ _kerberos._udp.example.com has SRV record 0 100 88 dc.example.com
       the domain name.</para>
   
     <para>There must be a valid Kerberos host key for the server in the <code>/etc/krb5.keytab</code>
-      file. It may be necessary to create a kerberos service principal and update the keytab if it
-      is not present. Depending on your domain type different service names are required:</para>
+      file. Alternatively, if you would like to use a different keytab, EI:
+      <code>/etc/httpd/conf/ipa.keytab</code>you can do so by linking it to
+      <code>/etc/cockpit/krb5.keytab</code>. It may be necessary to create a kerberos
+      service principal and update the keytab if it is not present. Depending on your
+      domain type different service names are required:</para>
```

Comment:
This diverges from Cockpit's design principles that there should be no cockpit-specific configuration. I would like to understand why it's reasonable to make an exception here - mostly, if someone wants to use ipa for authenticating users, why wouldn't they symlink that to `/etc/krb5.keytab` directly, so that it consistently works with VT/ssh/other logins as well?

Reply:
I don't think this diverges from Cockpit's design principles that much. The transport (auth, cert, etc) has always had configuration. eg: /etc/cockpit/ws-certs.d /etc/cockpit.conf WebService section, ...

Reply:
> if someone wants to use ipa for authenticating users, why wouldn't they symlink that to /etc/krb5.keytab directly, so that it consistently works with VT/ssh/other logins as well?

The use case here is when you need the HTTP/host principal to be available to both cockpit and something else ei (apache). Often that something else requires special permissions and location of it's keytab file.

I see this as similar to the fact that cockpit-ws needs keys and certificates for https.